spec/classes/core/perfsonar_spec.rb

```diff
@@ -6,7 +6,7 @@
   on_supported_os.each do |os, os_facts|
     context "on #{os}" do
       let(:facts) { os_facts }
-      let(:fqdn) { facts[:fqdn] }
+      let(:fqdn) { facts[:networking]['fqdn'] }
```

Comment:
this let shouldn't be required because rspec-puppet-fact sets the fqdn properly.

Reply:
This might be a left over from the past... at one point `fqdn` (not a fact lookup) was used in some examples.

Reply:
It wasn't needed anymore. It has been removed.